src/unix/spawn-helper.cc

```diff
@@ -0,0 +1,63 @@
+#include <fcntl.h>
+#include <signal.h>
+#include <string>
```

Comment:
Linux/gcc additionally needs `#include <string.h>` here, otherwise it will not compile...